SPECS/mariner-rpm-macros/macros

```diff
@@ -43,9 +43,7 @@
 %__os_install_post    \
     %{_rpmconfigdir}/brp-compress \
     %{_rpmconfigdir}/brp-strip %{__strip} \
-    %{_rpmconfigdir}/brp-strip-debug-symbols %{__strip} \
```

Comment:
Idea: what do you think about bundling the three macro declarations mentioned in [this article](https://imvoid.wordpress.com/2013/06/05/stripping-and-stopping-stripping-of-binaries-in-rpm-build/) into a new macro. Something like `%no_debuginfo_keep_symbols`. I haven't checked if the method suggested by the articles works, but the title looks promissing.;)

Reply:
That's an interesting idea, although wouldn't this require updating every affected SPEC? (I believe there are over 100 with this issue). This would also require changes from derivative teams who build with our toolkit.